mention Dev desktops, to help with LLVM builds (#1613)
rust-lang/rustc-dev-guide: src/backend/updating-llvm.md

Ideally the above instructions are pretty smooth, but here's some caveats to
keep in mind while going through them:

* LLVM bugs are hard to find, don't hesitate to ask for help!
  Bisection is definitely your friend here
  (yes LLVM takes forever to build, yet bisection is still your friend).
  Note that you can make use of [Dev Desktops],

>   an initiative to provide remote access to powerful hardware to developers of Rust.

Reviewer: ```suggestion
  which is an initiative to provide the contributors with remote access to powerful hardware.
```
